automation/taskcluster/androidTest/flank-arm64-v8a.yml

```diff
@@ -19,8 +19,7 @@ gcloud:
   performance-metrics: true
 
   test-targets:
-    - notPackage org.mozilla.fenix.screenshots
-    - notPackage org.mozilla.fenix.syncintegration
+  - class org.mozilla.fenix.ui.CollectionTest
```

Comment:
Don't forget to rollback the changes in this file.

Reply:
Thank you! Will drop that commit.